find-billy.desktop.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/find-billy.desktop b/find-billy.desktop
+index e5d1f38..fef56d6 100644
+--- a/find-billy.desktop
++++ b/find-billy.desktop
+@@ -13,8 +13,8 @@ Comment=Play a 2D platform game
+ Comment[en]=Play a 2D platform game
+ Comment[de]=Spiele ein 2D-Plattformspiel
+ Comment[fr]=Jouez à un jeu de plateforme en 2D 
+-Icon=find-billy
+-Exec=find-billy
++Icon=eu.annaaurora.find_billy.Find_Billy
++Exec=eu.annaaurora.find_billy.Find_Billy
```

Comment:
this exec doesn't exist.

Reply:
it's not clear to me what the executable is called with org.godotengine.godot.BaseApp